src/components/Button/Button.json

```diff
@@ -1,16 +1,6 @@
 {
   "name": "Button",
-  "template": "Button.html",
-  "class": "ms-Button",
-  "wrapBranches": true,
-  "fileOrder": [
-    "Button.html",
-    "Button.Primary.html",
-    "Button.Hero.html",
-    "Button.Compound.html",
-    "Button.Submenu.html"
-  ],
-  "props":  {
+  "props": {
     "label": "Create Account",
```

Comment:
@battletoilet: All of the props should be coming from the documentation files, right? When I remove these props entirely I get build errors. The same goes for CommandButton, MessageBanner, and SearchBox.